inventory/host_vars/www.coopcircuits.fr/config.yml

```diff
@@ -25,3 +25,4 @@ enable_nginx_logging: true
 enable_rails_apm: "true" # has to be explicitly defined as a string due to some datadog bug
 
 attachment_path: "home/openfoodnetwork/apps/openfoodnetwork/current/public/spree/products/:id/:style/:basename.:extension"
+attachment_url: ofn-prod.s3.us-east-1.amazonaws
```

Comment:
@luisramos0 is this a typo? Shouldn't `.amazonaws` be `.amazonaws.com` ..? :grimacing:

Reply:
![eagle-eye](https://media.giphy.com/media/SLD8eKFPuUDuw/giphy.gif)

Reply:
lol yes typo, well spotted!